packages/docs/page-config/ui-elements/pagination/examples/LimitVisible.vue

```diff
@@ -3,6 +3,7 @@
     v-model="value"
     :pages="15"
     :visible-pages="4"
+    class="justify-center sm:justify-start"
```

Comment:
It still overflows in some cases, but it looks so much better than before. I think we need to make paddings smaller on mobile screen and add scroll to example card. But it is in a different issue!